Common/config/rule.hpp

```diff
@@ -0,0 +1,60 @@
+#include "taskbarappearance.hpp"
+
+struct Rule : TaskbarAppearance
+{
+	std::wstring m_Class;
+	std::wstring m_Title;
+	std::wstring m_ProcessName;
+
+	constexpr Rule() noexcept = default;
+	constexpr Rule(std::wstring m_class, std::wstring m_title, std::wstring m_processName, ACCENT_STATE accent, Util::Color color, bool showPeek) noexcept :
```

Comment:
No `m_` for parameters, this constructor is not noexcept (std::wstring's constructor can throw)